cg/apps/demultiplex/demultiplex_api.py

```diff
@@ -106,7 +106,7 @@ def get_stdout_logfile(flow_cell: FlowCell) -> Path:
         """Create the path to the stdout logfile."""
         return Path(flow_cell.path, f"{DemultiplexingAPI.get_run_name(flow_cell)}.stdout")
 
-    def get_all_demultiplex_flow_cells_out_dirs(self) -> List[Path]:
+    def get_all_flow_cells_dirs(self) -> List[Path]:
```

Comment:
```suggestion
    def get_all_demultiplexed_flow_cells_dirs(self) -> List[Path]:
```

Reply:
think demultiplexed is good here since we have two positions to which we refer to flow cell dirs (in and out)